packages/pipeline/src/data_sources/contract-wrappers/utils.ts

```diff
@@ -0,0 +1,60 @@
+import { DecodedLogArgs, LogWithDecodedArgs } from 'ethereum-types';
```

Comment:
Didn't see tests for these utils. They seem like core functionality which may be worth testing especially under atypical conditions like retryable and non-retryable errors.

Reply:
👍 Good point. I just added tests in the most recent commit.